better-font-awesome.php

```diff
@@ -380,7 +389,7 @@ public function create_admin_page() {
 	 */
 	public function add_settings() {
 		register_setting(
-			'option_group', // Option group.
+			self::SLUG, // Option group.
```

Comment:
Same comment as above re: changing an existing settings group name.

Reply:
Same as above. The intention of this was to be prepared in case the Javascript file doesn't load and WordPress saves the settings the default way, to have WordPress use the right options name.